### PR TITLE
cleanup build

### PR DESCRIPTION
**Pull request by jamesward**
use consistent conventions

### CODE REVIEW

File: examples/build.gradle [hunk 1]
```diff
@@ -21,30 +21,29 @@ repositories {
...
+apply plugin: "application"
 
 dependencies {
   // This repo
-  implementation "io.grpc:grpc-kotlin-stub:${grpc_kotlin_version}"
+  implementation "io.grpc:grpc-kotlin-stub:$grpc_kotlin_version"
 
-  implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.61"
-  implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:${coroutines_version}"
+  implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
```
Comment (chalin) [inline examples/build.gradle:38]: No version for `kotlin:kotlin-stdlib-jdk8`?

Comment (jamesward) [reply examples/build.gradle:38]: The plugin fills it in based on the Kotlin version.

Comment (chalin) [reply examples/build.gradle:38]: Might it make sense to add a comment to that effect, or is it too much common knowledge?

**Comment by chalin** [comment]
FYI, Travis reports failures.

File: examples/build.gradle (hunk 1 above)
Comment (jamesward) [reply examples/build.gradle:38]: The official docs for Kotlin use this convention and it's common practice. So I think we should be good.

**Comment by lowasser** [comment]
I think those failures are unrelated?

**Pull request merged**